beetsplug/replaygain.py

```diff
@@ -232,6 +267,12 @@ def compute_chunk_gain(self, items, is_album):
 
         self._log.debug(u'analysis finished: {0}', output)
         results = self.parse_tool_output(output, path_list, is_album)
+
+        if gain_adjustment != 0:
```

Comment:
Zero integers are falsy in Python:
```suggestion
        if gain_adjustment:
```